### ISSUE TITLE
Display of ratio of future BGCs

### ISSUE THREAD

**Issue by Vcomea** [bcgov/CCISS_Review#29]
Ratio of Model Predictions of future BGCs
When many BGCs are predicted the legend overlaps with the bar chart

E.g. from SBSmc2
![image](https://github.com/user-attachments/assets/b1e9b33a-581d-45c3-8c6a-d47955bb695a)

**Comment by kdaust** [comment]
@Vcomea agreed - what do you suggest? Remove the legend entirely since the labels are already written on the chart?

**Comment by Vcomea** [comment]
@kdaust is it possible to shift the graph down so that it always appears directly below the legend? If not, maybe the legend is not needed.
